File: app/src/main/java/ru/mamykin/foreignbooksreader/models/Translation.java
package ru.mamykin.foreignbooksreader.models;

/**
 * Creation date: 5/29/2017
 * Creation time: 11:39 AM
 * @author Andrey Mamykin(mamykin_av)
 */
public class Translation {
    private int code;
    private String lang;
    private String[] text;

    public Translation() {
    }

    public Translation(int code, String lang, String[] text) {
        this.code = code;
        this.lang = lang;
        this.text = text;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getLang() {
        return lang;
    }

    public void setLang(String lang) {
        this.lang = lang;
    }

    public String[] getText() {
        return text;
    }

    public void setText(String[] text) {
        this.text = text;
    }
}

File: app/src/main/java/ru/mamykin/foreignbooksreader/models/Translation.kt
package ru.mamykin.foreignbooksreader.models

/**
 * Creation date: 5/29/2017
 * Creation time: 11:39 AM
 * @author Andrey Mamykin(mamykin_av)
 */
class Translation {
    var code: Int = 0
    var lang: String? = null
    var text: Array<String>? = null

    constructor() {}

    constructor(code: Int, lang: String, text: Array<String>) {
        this.code = code
        this.lang = lang
        this.text = text
    }
}